httpbinding/uri.go

```diff
@@ -78,3 +79,20 @@ func (u URIValue) BigDecimal(v *big.Float) error {
 	}
 	return u.modifyURI(v.Text('e', -1))
 }
+
+// ParseURI parses a Smithy HTTP binding trait URI
+func ParseURI(uri string) (path, query string) {
```

Comment:
Nit should this be `split` instead of parse since not really parsing the query string?